controller/src/test/java/io/pravega/controller/fault/ControllerClusterListenerTest.java

```diff
@@ -73,11 +72,13 @@
  */
 @Slf4j
 public class ControllerClusterListenerTest {
+
+    private static final RetryPolicy RETRY_POLICY = new ExponentialBackoffRetry(200, 10, 5000);
```

Comment:
This configuration is the same for all the classes, perhaps we can have the default retry value configured within the PravegaZkCuratorResource.

Reply:
Initially I configured it in PravegaZkCuratorResource , but there are some classes where a different RetryPolicy is being used.

For example: 

https://github.com/pravega/pravega/pull/5988/files#diff-cfab5bd3e94538550da7fda0f698ab12eed5cc90c9d0d7b77c2ce73c7f957480R94